composer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "neos/neos-development-collection",
+    "description": "Neos packages in a joined repository for pull requests.",
+    "type": "neos-package-collection",
+    "require": {
+        "typo3/flow": "*",
```

Comment:
I would like to merge this as is for now just becaue that reflects the current state of this branch. We can then think about adjusting the inner composer manifests as we like and update this merged one.